Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorWindow.cs

```diff
@@ -204,24 +204,6 @@ private InputActionAsset GetAssetFromDatabase()
 
         [SerializeField] private InputActionsEditorState m_State;
         [SerializeField] private string m_AssetGUID;
-
-        public static void SaveAsset(SerializedObject serializedAsset, InputActionsEditorWindow currentWindow = null)
-        {
-            if ((focusedWindow == null || focusedWindow is not InputActionsEditorWindow) && currentWindow == null)
```

Comment:
Is it ok this check is removed - I think Joao added this to make sure not used on the Input Asset flow where IMGUI still used. Perhaps its not really needed?